WorkflowConcurrency/Sources/AsyncSequenceWorker.swift

```diff
@@ -0,0 +1,69 @@
+import Foundation
+import Workflow
```

Comment:
have you tried silencing the concurrency warnings via use of Sendable requirements/annotations, or other means? since this is new, should we enforce that now (even though it's inconsistent with other API)?

Reply:
I did try `adding `@preconcurrency` like it suggested and it still gave the same warnings.
Could we enforce `Sendable` on `Output` and it still be convertible to AnyWorkflow since in Workflow Output is not Sendable?